contrib/ansible/deploy.yaml

```diff
@@ -0,0 +1,62 @@
+---
+- hosts: xsnippet
+  tasks:
+    - name: ensure python 2 is installed
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
+  gather_facts: false
+
+- hosts: xsnippet
+  vars:
+    xsnippet_image: xsnippet/xsnippet-api:latest
+  tasks:
+    - name: update apt cache
+      apt:  update_cache=yes cache_valid_time=3600
+
+    - name: install essential packages
+      apt: "name={{ item }} state=latest"
+      with_items:
+        - apt-transport-https
```

Comment:
Yeah, most distros still use `http` mirrors by default. Their reasoning behind this is that you don't need to trust a mirror, you only need to verify integrity of the packages (which is done relying on GPG).

While that's a valid justification, I still don't see a reason for them *not to use* `https` now when certificates are free and don't add to the cost of infrastructure. I know for Arch Linux it's possible to choose https-only mirrors just in case, not sure if the same holds for Ubuntu.